src/runtime/nuxt-image-mixins.ts

```diff
@@ -68,13 +68,23 @@ export default {
       srcset: [],
       blurry: null,
       loading: false,
-      loaded: false
+      loaded: false,
+      imageW: 0,
+      imageH: 0
     }
   },
   async fetch () {
-    if (!this.legacy) {
-      this.blurry = await this.$img.lqip(this.src)
+    if (this.legacy) {
+      return
     }
+    const image = await this.$img.lqip(this.src)
+    if (!image) {
+      return
+    }
+
+    this.blurry = image.url
```

Comment:
Same here. We are using different terms for lqip while it can be simply called placeholder everywhere